presto-hive/src/main/java/com/facebook/presto/hive/HiveClientConfig.java

```diff
@@ -965,4 +967,17 @@ public boolean isTableStatisticsEnabled()
     {
         return tableStatisticsEnabled;
     }
+
+    @Config("hive.username")
```

Comment:
how does it relate to setting `HADOOP_USER_NAME`?, see https://prestodb.io/docs/current/connector/hive.html

Reply:
Hadoop has its own RPC protocol, the client sends UGI to the server. But Metastore is achieved through thrift, send UGI need to call set_ugi method.

Reply:
`HADOOP_USER_NAME` can only be set once, but there will be multiple Hive catalogs, different clusters, different users.